mods/ts/rules/gdi-support.yaml

```diff
@@ -35,6 +35,129 @@ GAGATE_B:
 	Tooltip:
 		Name: GDI Gate
 
+GAFSDF:
+	Inherits@WALL: ^Wall
+	Inherits@EMPDISABLE: ^EmpDisable
+	Inherits@IDISABLED: ^DisableOnLowPowerOrPowerDown
+	Buildable:
+		Queue: Defense
+		BuildPaletteOrder: 20
+		Prerequisites: gafire, ~structures.gdi
+		Description: Destroys units and blocks enemy fire when active.
+	-Building:
+	EnergyWall:
+		Dimensions: 1,1
+		Footprint: x
+		BuildSounds: place2.aud
+		Adjacent: 7
+		TerrainTypes: Clear, Rough, Road, DirtRoad, Green, Sand, Pavement
+		ActiveCondition: firestorm-active && !disabled
+		Weapon: Firestorm
+	SoundOnDamageTransition:
+		DamagedSounds:
+		DestroyedSounds:
+	Valued:
+		Cost: 50
+	CustomSellValue:
+		Value: 0
+	Tooltip:
+		Name: Firestorm Wall Section
+	Health:
+		HP: 200
+	Armor:
+		Type: Concrete
+	BlocksProjectiles:
+		RequiresCondition: firestorm-active && !disabled
+		Height: 9999
```

Comment:
This still has the issue of not blocking rockets. I'd really like to get to the bottom of this issue before merging.

Reply:
All TS missiles simply have `Blockable: false`.
My guess would be that we'll have to replace the `Blockable` boolean by something like `BlockerTypes` and only let projectiles get blocked by their listed types. Bonus points for adding ´ValidStances` to `BlocksProjectiles` on top of that, so we could make firestorm only block enemy projectiles, if we wanted to.

Reply:
I thought the blocking heights was supposed to fix this?